content/news/2013/03/2013-03-21-american-red-cross-tornado-app.md

```diff
@@ -5,7 +5,6 @@ title: American Red Cross Tornado App
 summary: The American Red Cross Tornado App gets your family and home ready for a tornado. The app, which is available on the Android and iOS platforms, puts everything you need to know
 topics:
   - mobile
-  - american-red-cross
   - mobile-apps
   - mobile-gov
   - NOAA
```

Comment:
Another agency?

Reply:
Was removed from all files, added to list in initial comment